js/src/components/Sidebar.js

```diff
@@ -2,25 +2,32 @@
 
 import React from "react";
 import PropTypes from "prop-types";
+import { Provider as StoreProvider } from "react-redux";
 
 import SidebarItem from "./SidebarItem";
+import ReadabilityAnalysis from "./contentAnalysis/ReadabilityAnalysis";
 
 /**
  * Creates the Sidebar component.
  *
- * @param {bool} isContentAnalysisActive Whether or not the readability analysis is active or not.
- * @param {bool} isKeywordAnalysisActive Whether or not the readability analysis is active or not.
+ * @param {bool}   isContentAnalysisActive Whether or not the readability analysis is active or not.
+ * @param {bool}   isKeywordAnalysisActive Whether or not the readability analysis is active or not.
```

Comment:
The variable name let me think this is a bout the keyword analysis instead of the readability analysis